packages/coding-standards/.phpcs.xml.dist-sample

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0"?>
+<ruleset name="Jetpack-Plugin">
+	<config name="minimum_supported_wp_version" value="5.3" />
```

Comment:
No spaces expected before closing `/>` - this is XML, not XHTML. If you run xmllint against it, it will normalise it to have no-space.

Reply:
d02d394a2